recipe/0001-Adapt-for-Windows-build.patch

```diff
@@ -0,0 +1,58 @@
+From 091a3edbd59bb5601df2acffb698849fb807193d Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Julian=20R=C3=BCth?= <julian.rueth@fsfe.org>
```

Comment:
Keep both our name/email here?

Reply:
Sorry, I guess you're the original author.